validation/citrus-validation-hamcrest/src/main/java/org/citrusframework/validation/matcher/hamcrest/HamcrestValidationMatcher.java

```diff
@@ -500,7 +500,7 @@ private static class Tokenizer {
         /**
          * Regular expression with three alternative parts (ored) to match:
          * <ol>
-         *   <li> `(sometext)` - Quoted parameter block of a matcher.</li>
+         *   <li> (`sometext`) - Quoted parameter block of a matcher.</li>
```

Comment:
should be `('sometext')`, I think? because ` is not a "quote"...

Reply:
Indeed true.